File: javascript/1_multiple_3_5.js
var original = limit => {
  var list = Array.from(Array(limit).keys());
  var multiple_filter = n => {
    return (n % 3 === 0) || (n % 5 === 0);	
  };
  return list.filter(multiple_filter).reduce((a,b) => a + b);
}

var start = new Date().getTime();
var result = original(1000);
var end = new Date().getTime();
console.log(`Result: ${result} in ${end - start} milliseconds`);

File: elixir/1_multiple_3_5.exs
original = fn limit ->
  multiple? = fn n -> rem(n, 3) == 0 or rem(n, 5) == 0 end
  l = for n <- 0..(limit - 1), multiple?.(n), do: n
  Enum.reduce(l, &+/2)	
end

{time, result} = :timer.tc(fn -> original.(1000) end)
IO.puts "Result: #{result} in #{time} microseconds"